### ISSUE TITLE
Group of search results metadata can be exported to XML (if budget allows)

### ISSUE THREAD

**Issue by jillpe** [scientist-softserv/west-virginia-university#94]
## Summary
related to 
- #90 

WVU would like there users to be able to export metadata for a group of search results via the UI to XML. **This requirement is only if there is budget available. Please prioritize all other tickets first and check with Jill before starting this ticket**

## Acceptance Criteria
- [ ] As a user, I can export metadata for a group of search results via the UI to XML

Example from GBH. Ref #95 
![Image](https://github.com/scientist-softserv/west-virginia-university/assets/10081604/a793365d-5b20-4f0f-a02e-d4991ac63f21)

## Testing Instructions
- [ ] Initiate a search
- [ ] Click the Export dropdown and export XML
- [ ] Observe the XML is in the desired structure

**Comment by jillpe** [comment]
Low Priority - only if there is budget available

**Comment by jillpe** [comment]
blocked by 
- #90

**Comment by kirkkwang** [comment]
✅ pass xml was exported, can't attach but here are the contents

<details><summary>XML</summary>

```xml
<?xml version="1.0" encoding="UTF-8"?>
<items xmlns:dcterms="http://purl.org/dc/terms/" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:edm="http://www.europeana.eu/schemas/edm/">
  <item>
    <dcterms:provenance>Robert J. Dole Institute of Politics</dcterms:provenance>
    <dcterms:title>Dole and Goldwater shake hands</dcterms:title>
    <dcterms:date>1964</dcterms:date>
    <dcterms:created>1964</dcterms:created>
    <dcterms:creator>Unknown</dcterms:creator>
    <dcterms:rights>http://rightsstatements.org/vocab/NKC/1.0/</dcterms:rights>
    <dcterms:language>zxx</dcterms:language>
    <dcterms:temporal>88th (1963-1964)</dcterms:temporal>
    <dcterms:relation>Dole Photograph Collection, 1900-2011</dcterms:relation>
    <dcterms:isPartOf>Collection 012, Box 6, Folder 15</dcterms:isPartOf>
    <dcterms:source>https://dolearchivecollections.ku.edu/index.php?p=collections/controlcard&amp;id=47&amp;q=</dcterms:source>
    <dcterms:identifier>http://congressarchivesdev.lib.wvu.edu/record/xxph_006_015_002bx</dcterms:identifier>
    <edm:preview>https://dolearchives.omeka.net/exhibits/show/voices-from-the-big-first/item/323</edm:preview>
    <edm:isShownAt>https://dolearchives.omeka.net/exhibits/show/voices-from-the-big-first/item/323</edm:isShownAt>
    <dc:type>black-and-white photograph</dc:type>
    <dcterms:subject>Government Operations and Politics</dcterms:subject>
    <dcterms:contributor>Dole, Robert J., 1923-2021; Goldwater, Barry M. (Barry Morris), 1909-1998</dcterms:contributor>
    <dcterms:format>test; second test</dcterms:format>
    <dcterms:description>Congressman Bob Dole and Senator Barry Goldwater shaking hands with an American flag in the background.</dcterms:description>
    <dcterms:type>StillImage</dcterms:type>
  </item>
  <item>
    <dcterms:provenance>Robert J. Dole Institute of Politics</dcterms:provenance>
    <dcterms:title>Congressman Dole's Introduction Of Senator Barry Goldwater</dcterms:title>
    <dcterms:date>1964 May 12</dcterms:date>
    <dcterms:created>5/12/64</dcterms:created>
    <dcterms:creator>Dole, Robert J., 1923-2021</dcterms:creator>
    <dcterms:rights>http://rightsstatements.org/vocab/NoC-US/1.0/</dcterms:rights>
    <dcterms:language>eng</dcterms:language>
    <dcterms:temporal>88th (1963-1964)</dcterms:temporal>
    <dcterms:relation>Robert J. Dole Speeches Collection, 1958-1996</dcterms:relation>
    <dcterms:isPartOf>Collection 019, Box 1, Folder 40</dcterms:isPartOf>
    <dcterms:source>https://dolearchivecollections.ku.edu/index.php?p=collections/controlcard&amp;id=52&amp;q=</dcterms:source>
    <dcterms:identifier>http://congressarchivesdev.lib.wvu.edu/record/xxc019_001_040_allbx</dcterms:identifier>
    <edm:preview>https://dolearchives.omeka.net/exhibits/show/voices-from-the-big-first/item/279</edm:preview>
    <edm:isShownAt>https://dolearchives.omeka.net/exhibits/show/voices-from-the-big-first/item/279</edm:isShownAt>
    <dc:type>speech (document)</dc:type>
    <dcterms:subject>Government Operations and Politics</dcterms:subject>
    <dcterms:contributor>Dole, Robert J., 1923-2021; Goldwater, Barry M. (Barry Morris), 1909-1998</dcterms:contributor>
    <dcterms:format>test; second test</dcterms:format>
    <dcterms:description>Speech given by Congressman Robert J. Dole to introduce Senator Barry Goldwater at a rally in Madison Square Garden in New York City.</dcterms:description>
    <dcterms:type>Text</dcterms:type>
  </item>
  <item>
    <dcterms:provenance>Robert J. Dole Institute of Politics</dcterms:provenance>
    <dcterms:title>Buffalo NY: Rockefeller and Dole Speeches</dcterms:title>
    <dcterms:date>1976 September 16</dcterms:date>
    <dcterms:created>9/16/76</dcterms:created>
    <dcterms:creator>Unknown</dcterms:creator>
    <dcterms:rights>http://rightsstatements.org/vocab/CNE/1.0/</dcterms:rights>
    <dcterms:language>eng</dcterms:language>
    <dcterms:temporal>94th (1975-1976)</dcterms:temporal>
    <dcterms:relation>Robert J. Dole Audio Cassette Collection, 1966-1997</dcterms:relation>
    <dcterms:isPartOf>Collection 016, Cassette k001, Side A</dcterms:isPartOf>
    <dcterms:source>https://dolearchivecollections.ku.edu/index.php?p=collections/controlcard&amp;id=50&amp;q=</dcterms:source>
    <dcterms:identifier>http://congressarchivesdev.lib.wvu.edu/record/xxc016_k001_a.mp3bx</dcterms:identifier>
    <edm:preview>https://dolearchivecollections.ku.edu/collections/1976/assets/c016_k001_a.mp3</edm:preview>
    <edm:isShownAt>https://dolearchivecollections.ku.edu/collections/1976/resources/</edm:isShownAt>
    <dc:type>audio recording; speech (communication function)</dc:type>
    <dcterms:subject>Government Operations and Politics</dcterms:subject>
    <dcterms:contributor>Dole, Robert J., 1923-2021; Rockefeller, Nelson A. (Nelson Aldrich), 1908-1979</dcterms:contributor>
    <dcterms:format>test; second test</dcterms:format>
    <dcterms:description>Large crowd, Rally for Republican ticket, Stump Speech on Ford &amp; Rockefeller, NY State GOP leaders</dcterms:description>
    <dcterms:type>Text</dcterms:type>
  </item>
  <item>
    <dcterms:provenance>Robert J. Dole Institute of Politics</dcterms:provenance>
    <dcterms:title>Folder 6: D.C. Republican Women - Washington, D.C., 7 February 1977</dcterms:title>
    <dcterms:date>1977 February 7</dcterms:date>
    <dcterms:created>2/7/77</dcterms:created>
    <dcterms:creator>Dole, Robert J., 1923-2021</dcterms:creator>
    <dcterms:rights>http://rightsstatements.org/vocab/NoC-US/1.0/</dcterms:rights>
    <dcterms:language>eng</dcterms:language>
    <dcterms:temporal>95th (1977-1978)</dcterms:temporal>
    <dcterms:relation>Robert J. Dole Speeches Collection, 1958-1996</dcterms:relation>
    <dcterms:isPartOf>Collection 019, Box 17, Folder 6</dcterms:isPartOf>
    <dcterms:source>https://dolearchivecollections.ku.edu/index.php?p=collections/controlcard&amp;id=52&amp;q=</dcterms:source>
    <dcterms:identifier>http://congressarchivesdev.lib.wvu.edu/record/xxc019_017_006_allbx</dcterms:identifier>
    <edm:preview>https://dolearchivecollections.ku.edu/collections/speeches/017/c019_017_006_all.pdf</edm:preview>
    <edm:isShownAt>https://dolearchivecollections.ku.edu/index.php?p=collections/findingaid&amp;id=52&amp;q=</edm:isShownAt>
    <dc:type>speech (document)</dc:type>
    <dcterms:subject>Economics and Public Finance</dcterms:subject>
    <dcterms:contributor>Dole, Robert J., 1923-2021</dcterms:contributor>
    <dcterms:format>test; second test</dcterms:format>
    <dcterms:description>Folder 6: D.C. Republican Women - Washington, D.C., 7 February 1977</dcterms:description>
    <dcterms:type>Text</dcterms:type>
  </item>
  <item>
    <dcterms:provenance>Robert J. Dole Institute of Politics</dcterms:provenance>
    <dcterms:title>Senator Bob Dole addressing farmers at Senate Agriculture Committee Hearings, 1979</dcterms:title>
    <dcterms:date>1979</dcterms:date>
    <dcterms:created>1979</dcterms:created>
    <dcterms:creator>Unknown</dcterms:creator>
    <dcterms:rights>https://creativecommons.org/publicdomain/mark/1.0/</dcterms:rights>
    <dcterms:language>zxx</dcterms:language>
    <dcterms:temporal>96th  (1979–1981); 97th  (1981–1983)</dcterms:temporal>
    <dcterms:relation>Dole Photograph Collection (Old Arrangement - For Reference Only), 1880-1996</dcterms:relation>
    <dcterms:isPartOf>Collection 36719, Box 45: A [1], Folder 704</dcterms:isPartOf>
    <dcterms:source>https://dolearchivecollections.ku.edu/index.php?p=collections/controlcard&amp;id=9</dcterms:source>
    <dcterms:identifier>http://congressarchivesdev.lib.wvu.edu/record/xxpla_1_102bx</dcterms:identifier>
    <edm:preview>https://dolearchivecollections.ku.edu/index.php?p=digitallibrary/digitalcontent&amp;id=1607</edm:preview>
    <edm:isShownAt/>
    <dc:type>black-and-white photograph</dc:type>
    <dcterms:format>test; second test</dcterms:format>
    <dcterms:description/>
    <dcterms:type>StillImage</dcterms:type>
  </item>
  <item>
    <dcterms:provenance>Robert J. Dole Institute of Politics</dcterms:provenance>
    <dcterms:title>Ronald Reagan to Bob Dole, 1981-12-23</dcterms:title>
    <dcterms:date>1981 December 23</dcterms:date>
    <dcterms:created>12/23/81</dcterms:created>
    <dcterms:creator>Reagan, Ronald</dcterms:creator>
    <dcterms:rights>http://rightsstatements.org/vocab/CNE/1.0/</dcterms:rights>
    <dcterms:language>eng</dcterms:language>
    <dcterms:temporal>97th  (1981–1983)</dcterms:temporal>
    <dcterms:relation>VIP Letters, 1960-2008</dcterms:relation>
    <dcterms:isPartOf>Collection 020, Box 12, Item 22</dcterms:isPartOf>
    <dcterms:source>https://dolearchivecollections.ku.edu/index.php?p=collections/findingaid&amp;id=55&amp;q=</dcterms:source>
    <dcterms:identifier>http://congressarchivesdev.lib.wvu.edu/record/xxc020_012_000_022bx</dcterms:identifier>
    <edm:preview>https://dolearchivecollections.ku.edu/collections/vip_letters/c020_012_000_022.pdf</edm:preview>
    <edm:isShownAt>https://dolearchivecollections.ku.edu/index.php?p=collections/findingaid&amp;id=55&amp;q=&amp;rootcontentid=162379#id162379</edm:isShownAt>
    <dc:type>correspondence</dc:type>
    <dcterms:contributor>Dole, Robert J., 1923-2021; Reagan, Ronald</dcterms:contributor>
    <dcterms:format>test; second test</dcterms:format>
    <dcterms:description>Letter from President of the United States Ronald Reagan to Senator Bob Dole thanking him for agreeing to serve on the new National Commission on Social Security Reform.</dcterms:description>
    <dcterms:type>Text</dcterms:type>
  </item>
  <item>
    <dcterms:provenance>Robert J. Dole Institute of Politics</dcterms:provenance>
    <dcterms:title>Ronald Reagan Signing Social Security Act Outside</dcterms:title>
    <dcterms:date>1983 April 18</dcterms:date>
    <dcterms:created>4/18/83</dcterms:created>
    <dcterms:creator>Unknown; Still unknown</dcterms:creator>
    <dcterms:rights>http://rightsstatements.org/vocab/CNE/1.0/</dcterms:rights>
    <dcterms:language>zxx</dcterms:language>
    <dcterms:temporal>98th  (1983–1985)</dcterms:temporal>
    <dcterms:relation>Dole Photograph Collection, 1900-2011</dcterms:relation>
    <dcterms:isPartOf>Collection 012, Box 54, Folder 2</dcterms:isPartOf>
    <dcterms:source>https://dolearchivecollections.ku.edu/index.php?p=collections/findingaid&amp;id=47&amp;q=</dcterms:source>
    <dcterms:identifier>http://congressarchivesdev.lib.wvu.edu/record/xxph_054_012_002bx</dcterms:identifier>
    <edm:preview>https://dolearchivecollections.ku.edu/index.php?p=digitallibrary/digitalcontent&amp;id=286</edm:preview>
    <edm:isShownAt>https://dolearchivecollections.ku.edu/index.php?p=collections/findingaid&amp;id=47&amp;q=</edm:isShownAt>
    <dc:type>color photograph</dc:type>
    <dcterms:format>test; second test</dcterms:format>
    <dcterms:description>Kansas Senator Bob Dole, Representative Claude Pepper, Senator Daniel P. Moynihan, Representative Barber Conable, and House Speaker "Tip" O'Neill watch as President Reagan signs the "Social Security Amendments of 1983" into law.</dcterms:description>
    <dcterms:type>StillImage</dcterms:type>
  </item>
  <item>
    <dcterms:provenance>Robert J. Dole Institute of Politics</dcterms:provenance>
    <dcterms:title>Dole at White House Social Security Signing Session Hails Bipartisan Package, 4/20/1983</dcterms:title>
    <dcterms:date>1983 April 20</dcterms:date>
    <dcterms:created>4/20/83</dcterms:created>
    <dcterms:creator>Dole, Robert J., 1923-2021</dcterms:creator>
    <dcterms:rights>http://rightsstatements.org/vocab/CNE/1.0/</dcterms:rights>
    <dcterms:language>eng</dcterms:language>
    <dcterms:temporal>97th  (1981–1983)</dcterms:temporal>
    <dcterms:relation>Digitized Press Releases, 1961-1996</dcterms:relation>
    <dcterms:isPartOf>Collection 015, Box 25, Folder 16</dcterms:isPartOf>
    <dcterms:source>https://dolearchivecollections.ku.edu/index.php?p=collections/findingaid&amp;id=40&amp;q=</dcterms:source>
    <dcterms:identifier>http://congressarchivesdev.lib.wvu.edu/record/xxs-press_025_016_020bx</dcterms:identifier>
    <edm:preview>https://dolearchivecollections.ku.edu/collections/press_releases/s-press_025_016_020.pdf</edm:preview>
    <edm:isShownAt>https://dolearchivecollections.ku.edu/index.php?p=collections/findingaid&amp;id=40&amp;rootcontentid=191544&amp;q=social+security#id191639</edm:isShownAt>
    <dc:type>press release</dc:type>
    <dcterms:format>test; second test</dcterms:format>
    <dcterms:description/>
    <dcterms:type>Text</dcterms:type>
  </item>
  <item>
    <dcterms:provenance>Robert J. Dole Institute of Politics</dcterms:provenance>
    <dcterms:title>Ronald Reagan to Bob Dole, 1983-04-26</dcterms:title>
    <dcterms:date>1983 April 26</dcterms:date>
    <dcterms:created>4/26/83</dcterms:created>
    <dcterms:creator>Reagan, Ronald</dcterms:creator>
    <dcterms:rights>http://rightsstatements.org/vocab/CNE/1.0/</dcterms:rights>
    <dcterms:language>eng</dcterms:language>
    <dcterms:temporal>97th  (1981–1983)</dcterms:temporal>
    <dcterms:relation>VIP Letters, 1960-2008</dcterms:relation>
    <dcterms:isPartOf>Collection 020, Box 12, Item 43</dcterms:isPartOf>
    <dcterms:source>https://dolearchivecollections.ku.edu/index.php?p=collections/findingaid&amp;id=55&amp;q=</dcterms:source>
    <dcterms:identifier>http://congressarchivesdev.lib.wvu.edu/record/xxc020_012_000_043bx</dcterms:identifier>
    <edm:preview>https://dolearchivecollections.ku.edu/collections/vip_letters/c020_012_000_043.pdf</edm:preview>
    <edm:isShownAt>https://dolearchivecollections.ku.edu/index.php?p=collections/findingaid&amp;id=55&amp;q=&amp;rootcontentid=162379#id162379</edm:isShownAt>
    <dc:type>correspondence</dc:type>
    <dcterms:contributor>Dole, Robert J., 1923-2021; Reagan, Ronald</dcterms:contributor>
    <dcterms:format>test; second test</dcterms:format>
    <dcterms:description>Letter from President of the United States Ronald Reagan to Senator Bob Dole thanking him for support of the Senate Concurrent Resolution 26, to deploy 100 peacekeepers missiles and develop a new single-warhead ICBM.</dcterms:description>
    <dcterms:type>Text</dcterms:type>
  </item>
  <item>
    <dcterms:provenance>Robert J. Dole Institute of Politics</dcterms:provenance>
    <dcterms:title>Statement of Senator Dole on S. 1 "the Social Security Amendments of 1983"</dcterms:title>
    <dcterms:date>1983 January 26</dcterms:date>
    <dcterms:created>1/26/83</dcterms:created>
    <dcterms:creator>Dole, Robert J., 1923-2021</dcterms:creator>
    <dcterms:rights>http://rightsstatements.org/vocab/CNE/1.0/</dcterms:rights>
    <dcterms:language>eng</dcterms:language>
    <dcterms:temporal>97th  (1981–1983)</dcterms:temporal>
    <dcterms:relation>Digitized Press Releases, 1961-1996</dcterms:relation>
    <dcterms:isPartOf>Collection 015, Box 25, Folder 13</dcterms:isPartOf>
    <dcterms:source>https://dolearchivecollections.ku.edu/index.php?p=collections/findingaid&amp;id=40&amp;q=</dcterms:source>
    <dcterms:identifier>http://congressarchivesdev.lib.wvu.edu/record/xxs-press_025_013_018bx</dcterms:identifier>
    <edm:preview>https://dolearchivecollections.ku.edu/collections/press_releases/s-press_025_013_018.pdf</edm:preview>
    <edm:isShownAt>https://dolearchivecollections.ku.edu/index.php?p=collections/findingaid&amp;id=40&amp;rootcontentid=191544&amp;q=social+security#id191567</edm:isShownAt>
    <dc:type>press release</dc:type>
    <dcterms:format>test; second test</dcterms:format>
    <dcterms:description/>
    <dcterms:type>Text</dcterms:type>
  </item>
  <item>
    <dcterms:provenance>Robert J. Dole Institute of Politics</dcterms:provenance>
    <dcterms:title>"Social Security Rescued" Press Release and Newsletter</dcterms:title>
    <dcterms:date>1983 June</dcterms:date>
    <dcterms:created>1983-06</dcterms:created>
    <dcterms:creator>Dole, Robert J., 1923-2021</dcterms:creator>
    <dcterms:rights>https://creativecommons.org/publicdomain/mark/1.0/</dcterms:rights>
    <dcterms:language>eng</dcterms:language>
    <dcterms:temporal>98th (1983-1985)</dcterms:temporal>
    <dcterms:relation>Digitized Press Releases, 1961-1996</dcterms:relation>
    <dcterms:isPartOf>Collection 015, Box 26, Folder 2</dcterms:isPartOf>
    <dcterms:source>https://dolearchivecollections.ku.edu/index.php?p=collections/controlcard&amp;id=40&amp;q=</dcterms:source>
    <dcterms:identifier>http://congressarchivesdev.lib.wvu.edu/record/xxs-press_026_002_001bx</dcterms:identifier>
    <edm:preview>https://dolearchivecollections.ku.edu/collections/press_releases/s-press_026_002_001.pdf</edm:preview>
    <edm:isShownAt>https://dolearchivecollections.ku.edu/index.php?p=collections/findingaid&amp;id=40&amp;q=</edm:isShownAt>
    <dc:type>press release; newsletter</dc:type>
    <dcterms:subject>Social Welfare</dcterms:subject>
    <dcterms:contributor>Dole, Robert J., 1923-2021</dcterms:contributor>
    <dcterms:format>test; second test</dcterms:format>
    <dcterms:description/>
    <dcterms:type>Text</dcterms:type>
  </item>
  <item>
    <dcterms:provenance>Robert J. Dole Institute of Politics</dcterms:provenance>
    <dcterms:title>Ronald Reagan to Bob Dole, 1983-03-05</dcterms:title>
    <dcterms:date>1983 March 5</dcterms:date>
    <dcterms:created>3/5/83</dcterms:created>
    <dcterms:creator>Reagan, Ronald</dcterms:creator>
    <dcterms:rights>http://rightsstatements.org/vocab/CNE/1.0/</dcterms:rights>
    <dcterms:language>eng</dcterms:language>
    <dcterms:temporal>97th  (1981–1983)</dcterms:temporal>
    <dcterms:relation>VIP Letters, 1960-2008</dcterms:relation>
    <dcterms:isPartOf>Collection 020, Box 12, Item 42</dcterms:isPartOf>
    <dcterms:source>https://dolearchivecollections.ku.edu/index.php?p=collections/findingaid&amp;id=55&amp;q=</dcterms:source>
    <dcterms:identifier>http://congressarchivesdev.lib.wvu.edu/record/xxc020_012_000_042bx</dcterms:identifier>
    <edm:preview>https://dolearchivecollections.ku.edu/collections/vip_letters/c020_012_000_042.pdf</edm:preview>
    <edm:isShownAt>https://dolearchivecollections.ku.edu/index.php?p=collections/findingaid&amp;id=55&amp;q=&amp;rootcontentid=162379#id162379</edm:isShownAt>
    <dc:type>correspondence</dc:type>
    <dcterms:contributor>Dole, Robert J., 1923-2021; Reagan, Ronald</dcterms:contributor>
    <dcterms:format>test; second test</dcterms:format>
    <dcterms:description>Letter from President of the United States Ronald Reagan to Senator Bob Dole giving Dole the signing pen from H.R. 1900, the Social Security Amendments of 1983.</dcterms:description>
    <dcterms:type>Text</dcterms:type>
  </item>
  <item>
    <dcterms:provenance>Robert J. Dole Institute of Politics</dcterms:provenance>
    <dcterms:title>Senator Bob Dole welcoming Queen Elizabeth II to the Capitol</dcterms:title>
    <dcterms:date>1991</dcterms:date>
    <dcterms:created>1991</dcterms:created>
    <dcterms:creator>United States. Congress. Senate.</dcterms:creator>
    <dcterms:rights>https://creativecommons.org/publicdomain/mark/1.0/</dcterms:rights>
    <dcterms:language>zxx</dcterms:language>
    <dcterms:temporal>102nd (1991-1993)</dcterms:temporal>
    <dcterms:relation>Dole Photograph Collection, 1900-2011</dcterms:relation>
    <dcterms:isPartOf>Collection 012, Box 35, Folder 31</dcterms:isPartOf>
    <dcterms:source>https://dolearchivecollections.ku.edu/index.php?p=collections/findingaid&amp;id=47&amp;q=</dcterms:source>
    <dcterms:identifier>http://congressarchivesdev.lib.wvu.edu/record/xxph_035_041_001bx</dcterms:identifier>
    <edm:preview>https://dolearchivecollections.ku.edu/index.php?p=digitallibrary/digitalcontent&amp;id=1648</edm:preview>
    <edm:isShownAt>https://dolearchivecollections.ku.edu/index.php?p=digitallibrary/digitalcontent&amp;id=1648</edm:isShownAt>
    <dc:type>color photograph</dc:type>
    <dcterms:subject>International Affairs</dcterms:subject>
    <dcterms:contributor>Dole, Robert J., 1923-2021; Elizabeth II, Queen of Great Britain, 1926-2022</dcterms:contributor>
    <dcterms:format>test; second test</dcterms:format>
    <dcterms:description>Senator Bob Dole shakes hands with Queen Elizabeth II as she arrives at the U.S. Capitol.</dcterms:description>
    <dcterms:type>StillImage</dcterms:type>
  </item>
  <item>
    <dcterms:provenance>Hawaiʻi Congressional Papers Collection, University of Hawaiʻi at Mānoa Library, Honolulu, HI</dcterms:provenance>
    <dcterms:title>Democratic Convention Speech</dcterms:title>
    <dcterms:date>1994</dcterms:date>
    <dcterms:created>1994</dcterms:created>
    <dcterms:creator>Inouye, Daniel K., 1924-2012</dcterms:creator>
    <dcterms:rights>http://rightsstatements.org/vocab/NKC/1.0/</dcterms:rights>
    <dcterms:language>eng</dcterms:language>
    <dcterms:temporal>103rd (1993-1995)</dcterms:temporal>
    <dcterms:relation>Senator Daniel K. Inouye Papers</dcterms:relation>
    <dcterms:isPartOf>MANUSCRIPT-HCPC00011, Speeches, Box SP10, Folder 47</dcterms:isPartOf>
    <dcterms:source>https://archivesspace.library.manoa.hawaii.edu/repositories/4/resources/15</dcterms:source>
    <dcterms:identifier>http://congressarchivesdev.lib.wvu.edu/record/10524/63469</dcterms:identifier>
    <edm:preview>https://evols.library.manoa.hawaii.edu/bitstreams/b85fbac0-c2e1-4ea9-9785-ba500a309a7e/download</edm:preview>
    <edm:isShownAt>https:/evols.library.manoa.hawaii.edu/items/486cb54e-9876-4107-8343-d1b3aea2a9e5</edm:isShownAt>
    <dc:type>speeches (documents)</dc:type>
    <dcterms:subject>Government Operations and Politics</dcterms:subject>
    <dcterms:description/>
    <dcterms:type>Text</dcterms:type>
  </item>
  <item>
    <dcterms:provenance>Hawaiʻi Congressional Papers Collection, University of Hawaiʻi at Mānoa Library, Honolulu, HI</dcterms:provenance>
    <dcterms:title>Democratic Convention Speech</dcterms:title>
    <dcterms:date>1994</dcterms:date>
    <dcterms:created>1994</dcterms:created>
    <dcterms:creator>Inouye, Daniel K., 1924-2012</dcterms:creator>
    <dcterms:rights>http://rightsstatements.org/vocab/NKC/1.0/</dcterms:rights>
    <dcterms:language>eng</dcterms:language>
    <dcterms:temporal>103rd (1993-1995)</dcterms:temporal>
    <dcterms:relation>Senator Daniel K. Inouye Papers</dcterms:relation>
    <dcterms:isPartOf>MANUSCRIPT-HCPC00011, Speeches, Box SP10, Folder 47</dcterms:isPartOf>
    <dcterms:source>https://archivesspace.library.manoa.hawaii.edu/repositories/4/resources/15</dcterms:source>
    <dcterms:identifier>http://congressarchivesdev.lib.wvu.edu/record/http://hdl.handle.net/10524/63469</dcterms:identifier>
    <edm:preview>https://evols.library.manoa.hawaii.edu/bitstreams/b85fbac0-c2e1-4ea9-9785-ba500a309a7e/download</edm:preview>
    <edm:isShownAt>https:/evols.library.manoa.hawaii.edu/items/486cb54e-9876-4107-8343-d1b3aea2a9e5</edm:isShownAt>
    <dc:type>speeches (documents)</dc:type>
    <dcterms:subject>Government Operations and Politics</dcterms:subject>
    <dcterms:format>42 Pages</dcterms:format>
    <dcterms:description/>
    <dcterms:type>Text</dcterms:type>
  </item>
  <item>
    <dcterms:provenance>Hawaiʻi Congressional Papers Collection, University of Hawaiʻi at Mānoa Library, Honolulu, HI</dcterms:provenance>
    <dcterms:title>December 7th Service at Hickam AFB</dcterms:title>
    <dcterms:date>1999</dcterms:date>
    <dcterms:created>1999</dcterms:created>
    <dcterms:creator>Inouye, Daniel K., 1924-2012</dcterms:creator>
    <dcterms:rights>http://rightsstatements.org/vocab/NKC/1.0/</dcterms:rights>
    <dcterms:language>eng</dcterms:language>
    <dcterms:temporal>106th (1999-2001)</dcterms:temporal>
    <dcterms:relation>Senator Daniel K. Inouye Papers</dcterms:relation>
    <dcterms:isPartOf>MANUSCRIPT-HCPC00011, Speeches, Box SP12, Folder 19</dcterms:isPartOf>
    <dcterms:source>https://archivesspace.library.manoa.hawaii.edu/repositories/4/resources/15</dcterms:source>
    <dcterms:identifier>http://congressarchivesdev.lib.wvu.edu/record/10524/63594</dcterms:identifier>
    <edm:preview>https://evols.library.manoa.hawaii.edu/bitstreams/41f834af-09ba-45c7-ab5d-0a32ee9d7be6/download</edm:preview>
    <edm:isShownAt>https:/evols.library.manoa.hawaii.edu/items/a785b31b-94e5-4a5d-9122-3125cd7d624a</edm:isShownAt>
    <dc:type>speeches (documents)</dc:type>
    <dcterms:subject>Social Sciences and History; Armed Forces and National Security</dcterms:subject>
    <dcterms:description/>
    <dcterms:type>Text</dcterms:type>
  </item>
  <item>
    <dcterms:provenance>Hawaiʻi Congressional Papers Collection, University of Hawaiʻi at Mānoa Library, Honolulu, HI</dcterms:provenance>
    <dcterms:title>December 7th Service at Hickam AFB</dcterms:title>
    <dcterms:date>1999</dcterms:date>
    <dcterms:created>1999</dcterms:created>
    <dcterms:creator>Inouye, Daniel K., 1924-2012</dcterms:creator>
    <dcterms:rights>http://rightsstatements.org/vocab/NKC/1.0/</dcterms:rights>
    <dcterms:language>eng</dcterms:language>
    <dcterms:temporal>106th (1999-2001)</dcterms:temporal>
    <dcterms:relation>Senator Daniel K. Inouye Papers</dcterms:relation>
    <dcterms:isPartOf>MANUSCRIPT-HCPC00011, Speeches, Box SP12, Folder 19</dcterms:isPartOf>
    <dcterms:source>https://archivesspace.library.manoa.hawaii.edu/repositories/4/resources/15</dcterms:source>
    <dcterms:identifier>http://congressarchivesdev.lib.wvu.edu/record/http://hdl.handle.net/10524/63594</dcterms:identifier>
    <edm:preview>https://evols.library.manoa.hawaii.edu/bitstreams/41f834af-09ba-45c7-ab5d-0a32ee9d7be6/download</edm:preview>
    <edm:isShownAt>https:/evols.library.manoa.hawaii.edu/items/a785b31b-94e5-4a5d-9122-3125cd7d624a</edm:isShownAt>
    <dc:type>speeches (documents)</dc:type>
    <dcterms:subject>Social Sciences and History; Armed Forces and National Security</dcterms:subject>
    <dcterms:format>26 Pages</dcterms:format>
    <dcterms:description/>
    <dcterms:type>Text</dcterms:type>
  </item>
  <item>
    <dcterms:provenance>Robert J. Dole Institute of Politics</dcterms:provenance>
    <dcterms:title>Senator Bob Dole Oral History, 2007-10-16</dcterms:title>
    <dcterms:date>2007 October 16</dcterms:date>
    <dcterms:created>10/16/07</dcterms:created>
    <dcterms:creator>Richard Norton Smith</dcterms:creator>
    <dcterms:rights>http://rightsstatements.org/vocab/CNE/1.0/</dcterms:rights>
    <dcterms:language>eng</dcterms:language>
    <dcterms:temporal>110th  (2007–2009)</dcterms:temporal>
    <dcterms:relation>Dole Institute Oral History Project, 2002-2009</dcterms:relation>
    <dcterms:isPartOf>Collection 018, Bob Dole Oral History from 2007-10-16</dcterms:isPartOf>
    <dcterms:source>https://dolearchivecollections.ku.edu/index.php?p=collections/findingaid&amp;id=51&amp;q=</dcterms:source>
    <dcterms:identifier>http://congressarchivesdev.lib.wvu.edu/record/xxdole_bob_2007-10-16bx</dcterms:identifier>
    <edm:preview>https://dolearchivecollections.ku.edu/collections/oral_history/mp3/dole_bob_2007-10-16.mp3</edm:preview>
    <edm:isShownAt>https://dolearchives.omeka.net/items/show/335</edm:isShownAt>
    <dc:type>oral history (discipline)</dc:type>
    <dcterms:format>test; second test</dcterms:format>
    <dcterms:description>In this 2007 oral history interview with Richard Norton Smith, Dole talks about Reagan-era issues, including supply-side economics, Social Security reform, the White House staff, Dole's role as Majority Leader in the Senate, and other topics.</dcterms:description>
    <dcterms:type>Sound</dcterms:type>
  </item>
  <item>
    <dcterms:provenance>Robert J. Dole Institute of Politics</dcterms:provenance>
    <dcterms:title>Kassebaum-Baker, Sen. Nancy, 4/16/2009</dcterms:title>
    <dcterms:date>2009 April 16</dcterms:date>
    <dcterms:created>4/16/09</dcterms:created>
    <dcterms:creator>Dole, Robert J., 1923-2021; Williams, Brien R.</dcterms:creator>
    <dcterms:rights>http://rightsstatements.org/vocab/CNE/1.0/</dcterms:rights>
    <dcterms:language>eng</dcterms:language>
    <dcterms:temporal>111th (2009-2010)</dcterms:temporal>
    <dcterms:relation>Dole Institute Oral History Project, 2002-2009</dcterms:relation>
    <dcterms:isPartOf>Collection 018, Nancy Kassebaum-Baker Oral History from 2009-04-16</dcterms:isPartOf>
    <dcterms:source>https://dolearchivecollections.ku.edu/index.php?p=collections/findingaid&amp;id=51&amp;q=</dcterms:source>
    <dcterms:identifier>http://congressarchivesdev.lib.wvu.edu/record/xxkassebaum-baker_nancy_2009-04-16bx</dcterms:identifier>
    <edm:preview>https://dolearchivecollections.ku.edu/collections/oral_history/mp3/kassebaum-baker_nancy_2009-04-16.mp3</edm:preview>
    <edm:isShownAt>https://dolearchives.omeka.net/items/show/234</edm:isShownAt>
    <dc:type>sound</dc:type>
    <dcterms:subject>Agriculture and Food, Health, International Affairs</dcterms:subject>
    <dcterms:contributor>Dole, Robert J., 1923-2021; Baker, Nancy Kassebaum, 1932-</dcterms:contributor>
    <dcterms:format>test; second test</dcterms:format>
    <dcterms:description>In this 2009 oral history interview with Brien R. Williams, Kassebaum-Baker talks about Senator Dole's work ethic and the issues of abortion, the Panama Canal, and wheat subsidies on the campaign trail.    Baker represented Kansas in the U.S. Senate (1978-1997). She was the first female senator not elected to a seat held by her husband nor appointed to fill out a deceased husband’s term.</dcterms:description>
    <dcterms:type>Sound</dcterms:type>
  </item>
  <item>
    <dcterms:provenance>Robert J. Dole Institute of Politics</dcterms:provenance>
    <dcterms:title>Kassebaum-Baker, Sen. Nancy, 4/16/2009</dcterms:title>
    <dcterms:date>2009 April 16</dcterms:date>
    <dcterms:created>4/16/09</dcterms:created>
    <dcterms:creator>Williams, Brien R.</dcterms:creator>
    <dcterms:rights>http://rightsstatements.org/vocab/CNE/1.0/</dcterms:rights>
    <dcterms:language>eng</dcterms:language>
    <dcterms:temporal>111th (2009-2010)</dcterms:temporal>
    <dcterms:relation>Dole Institute Oral History Project, 2002-2009</dcterms:relation>
    <dcterms:isPartOf>Collection 018, Nancy Kassebaum-Baker Oral History from 2009-04-16</dcterms:isPartOf>
    <dcterms:source>https://dolearchivecollections.ku.edu/index.php?p=collections/findingaid&amp;id=51&amp;q=</dcterms:source>
    <dcterms:identifier>http://congressarchivesdev.lib.wvu.edu/record/xxLSqyFpjhe3gbx</dcterms:identifier>
    <edm:preview>https://www.youtube.com/watch?v=LSqyFpjhe3g</edm:preview>
    <edm:isShownAt>https://dolearchives.omeka.net/items/show/234</edm:isShownAt>
    <dc:type>moving image</dc:type>
    <dcterms:subject>Agriculture and Food, Health, International Affairs</dcterms:subject>
    <dcterms:contributor>Dole, Robert J., 1923-2021; Baker, Nancy Kassebaum, 1932-</dcterms:contributor>
    <dcterms:format>test; second test</dcterms:format>
    <dcterms:description>In this 2009 oral history interview with Brien R. Williams, Kassebaum-Baker talks about Senator Dole's work ethic and the issues of abortion, the Panama Canal, and wheat subsidies on the campaign trail.    Baker represented Kansas in the U.S. Senate (1978-1997). She was the first female senator not elected to a seat held by her husband nor appointed to fill out a deceased husband’s term.</dcterms:description>
    <dcterms:type>MovingImage</dcterms:type>
  </item>
  <item>
    <dcterms:provenance>Robert J. Dole Institute of Politics</dcterms:provenance>
    <dcterms:title>Social Security Rescued, 1983</dcterms:title>
    <dcterms:date>1999; 1983; 2023</dcterms:date>
    <dcterms:created>1983</dcterms:created>
    <dcterms:creator>Dole, Robert J., 1923-2021</dcterms:creator>
    <dcterms:rights>http://rightsstatements.org/vocab/CNE/1.0/</dcterms:rights>
    <dcterms:language>eng</dcterms:language>
    <dcterms:temporal>97th  (1981–1983)</dcterms:temporal>
    <dcterms:relation>Digitized Press Releases, 1961-1996</dcterms:relation>
    <dcterms:isPartOf>Collection 015, Box 25, Folder 15</dcterms:isPartOf>
    <dcterms:source>https://dolearchivecollections.ku.edu/index.php?p=collections/findingaid&amp;id=40&amp;q=</dcterms:source>
    <dcterms:identifier>http://congressarchivesdev.lib.wvu.edu/record/xxs-press_025_015_002bx</dcterms:identifier>
    <edm:preview>https://dolearchivecollections.ku.edu/collections/press_releases/s-press_025_015_002.pdf</edm:preview>
    <edm:isShownAt>https://dolearchivecollections.ku.edu/index.php?p=collections/findingaid&amp;id=40&amp;rootcontentid=191544&amp;q=social+security#id191549</edm:isShownAt>
    <dc:type>press release; newsletter</dc:type>
    <dcterms:format>test; second test</dcterms:format>
    <dcterms:description/>
    <dcterms:type>Text</dcterms:type>
  </item>
  <item>
    <dcterms:provenance>Hawaiʻi Congressional Papers Collection, University of Hawaiʻi at Mānoa Library, Honolulu, HI</dcterms:provenance>
    <dcterms:title>S. 1988, Native American Diabetes Bill</dcterms:title>
    <dcterms:date>April 15, 1986</dcterms:date>
    <dcterms:created>1986-04-15</dcterms:created>
    <dcterms:creator>Inouye, Daniel K., 1924-2012</dcterms:creator>
    <dcterms:rights>http://rightsstatements.org/vocab/NKC/1.0/</dcterms:rights>
    <dcterms:language>eng</dcterms:language>
    <dcterms:temporal>99th (1985-1987)</dcterms:temporal>
    <dcterms:relation>Senator Daniel K. Inouye Papers</dcterms:relation>
    <dcterms:isPartOf>MANUSCRIPT-HCPC00011, Speeches, Box SP7, Folder 56</dcterms:isPartOf>
    <dcterms:source>https://archivesspace.library.manoa.hawaii.edu/repositories/4/resources/15</dcterms:source>
    <dcterms:identifier>http://congressarchivesdev.lib.wvu.edu/record/10524/63351</dcterms:identifier>
    <edm:preview>https://evols.library.manoa.hawaii.edu/bitstreams/afaa1edc-a6e3-4cd4-98ff-47c7ee6ab88a/download</edm:preview>
    <edm:isShownAt>https:/evols.library.manoa.hawaii.edu/items/02535087-4cb7-4841-8749-e1bf3508020f</edm:isShownAt>
    <dc:type>speeches (documents)</dc:type>
    <dcterms:subject>Native Americans; Health</dcterms:subject>
    <dcterms:description/>
    <dcterms:type>Text</dcterms:type>
  </item>
  <item>
    <dcterms:provenance>Hawaiʻi Congressional Papers Collection, University of Hawaiʻi at Mānoa Library, Honolulu, HI</dcterms:provenance>
    <dcterms:title>S. 1988, Native American Diabetes Bill</dcterms:title>
    <dcterms:date>April 15, 1986</dcterms:date>
    <dcterms:created>1986-04-15</dcterms:created>
    <dcterms:creator>Inouye, Daniel K., 1924-2012</dcterms:creator>
    <dcterms:rights>http://rightsstatements.org/vocab/NKC/1.0/</dcterms:rights>
    <dcterms:language>eng</dcterms:language>
    <dcterms:temporal>99th (1985-1987)</dcterms:temporal>
    <dcterms:relation>Senator Daniel K. Inouye Papers</dcterms:relation>
    <dcterms:isPartOf>MANUSCRIPT-HCPC00011, Speeches, Box SP7, Folder 56</dcterms:isPartOf>
    <dcterms:source>https://archivesspace.library.manoa.hawaii.edu/repositories/4/resources/15</dcterms:source>
    <dcterms:identifier>http://congressarchivesdev.lib.wvu.edu/record/http://hdl.handle.net/10524/63351</dcterms:identifier>
    <edm:preview>https://evols.library.manoa.hawaii.edu/bitstreams/afaa1edc-a6e3-4cd4-98ff-47c7ee6ab88a/download</edm:preview>
    <edm:isShownAt>https:/evols.library.manoa.hawaii.edu/items/02535087-4cb7-4841-8749-e1bf3508020f</edm:isShownAt>
    <dc:type>speeches (documents)</dc:type>
    <dcterms:subject>Health; Native Americans</dcterms:subject>
    <dcterms:format>4 Pages</dcterms:format>
    <dcterms:description/>
    <dcterms:type>Text</dcterms:type>
  </item>
  <item>
    <dcterms:provenance>Hawaiʻi Congressional Papers Collection, University of Hawaiʻi at Mānoa Library, Honolulu, HI</dcterms:provenance>
    <dcterms:title>"Aloha Aina" A love for the land</dcterms:title>
    <dcterms:date>April 17, 1995</dcterms:date>
    <dcterms:created>1995-04-17</dcterms:created>
    <dcterms:creator>Inouye, Daniel K., 1924-2012</dcterms:creator>
    <dcterms:rights>http://rightsstatements.org/vocab/NKC/1.0/</dcterms:rights>
    <dcterms:language>eng</dcterms:language>
    <dcterms:temporal>103rd (1993-1995)</dcterms:temporal>
    <dcterms:relation>Senator Daniel K. Inouye Papers</dcterms:relation>
    <dcterms:isPartOf>MANUSCRIPT-HCPC00011, Speeches, Undated, Box SP1, Folder 18</dcterms:isPartOf>
    <dcterms:source>https://archivesspace.library.manoa.hawaii.edu/repositories/4/resources/15</dcterms:source>
    <dcterms:identifier>http://congressarchivesdev.lib.wvu.edu/record/10524/71492</dcterms:identifier>
    <edm:preview>https://evols.library.manoa.hawaii.edu/bitstreams/c5a30c06-5723-4dd2-bcda-550594c6986e/download</edm:preview>
    <edm:isShownAt>https:/evols.library.manoa.hawaii.edu/items/3cf13189-3d0e-43ba-9f80-97b02c2f1b48</edm:isShownAt>
    <dc:type>speeches (documents)</dc:type>
    <dcterms:subject>Native Americans; Housing and Community Development; Public Lands and Natural Resources; Arts, Culture, Religion</dcterms:subject>
    <dcterms:description/>
    <dcterms:type>Text</dcterms:type>
  </item>
  <item>
    <dcterms:provenance>Hawaiʻi Congressional Papers Collection, University of Hawaiʻi at Mānoa Library, Honolulu, HI</dcterms:provenance>
    <dcterms:title>"Aloha Aina" A love for the land</dcterms:title>
    <dcterms:date>April 17, 1995</dcterms:date>
    <dcterms:created>1995-04-17</dcterms:created>
    <dcterms:creator>Inouye, Daniel K., 1924-2012</dcterms:creator>
    <dcterms:rights>http://rightsstatements.org/vocab/NKC/1.0/</dcterms:rights>
    <dcterms:language>eng</dcterms:language>
    <dcterms:temporal>103rd (1993-1995)</dcterms:temporal>
    <dcterms:relation>Senator Daniel K. Inouye Papers</dcterms:relation>
    <dcterms:isPartOf>MANUSCRIPT-HCPC00011, Speeches, Undated, Box SP1, Folder 18</dcterms:isPartOf>
    <dcterms:source>https://archivesspace.library.manoa.hawaii.edu/repositories/4/resources/15</dcterms:source>
    <dcterms:identifier>http://congressarchivesdev.lib.wvu.edu/record/https://hdl.handle.net/10524/71492</dcterms:identifier>
    <edm:preview>https://evols.library.manoa.hawaii.edu/bitstreams/c5a30c06-5723-4dd2-bcda-550594c6986e/download</edm:preview>
    <edm:isShownAt>https:/evols.library.manoa.hawaii.edu/items/3cf13189-3d0e-43ba-9f80-97b02c2f1b48</edm:isShownAt>
    <dc:type>speeches (documents)</dc:type>
    <dcterms:subject>Arts, Culture, Religion; Public Lands and Natural Resources; Native Americans; Housing and Community Development</dcterms:subject>
    <dcterms:format>8 Pages</dcterms:format>
    <dcterms:description/>
    <dcterms:type>Text</dcterms:type>
  </item>
  <item>
    <dcterms:provenance>Hawaiʻi Congressional Papers Collection, University of Hawaiʻi at Mānoa Library, Honolulu, HI</dcterms:provenance>
    <dcterms:title>Kiwanis Club of Honolulu</dcterms:title>
    <dcterms:date>February 8, 1983</dcterms:date>
    <dcterms:created>1983-02-08</dcterms:created>
    <dcterms:creator>Inouye, Daniel K., 1924-2012</dcterms:creator>
    <dcterms:rights>http://rightsstatements.org/vocab/NKC/1.0/</dcterms:rights>
    <dcterms:language>eng</dcterms:language>
    <dcterms:temporal>98th (1983-1985)</dcterms:temporal>
    <dcterms:relation>Senator Daniel K. Inouye Papers</dcterms:relation>
    <dcterms:isPartOf>MANUSCRIPT-HCPC00011, Speeches, Chronological, Box SP5, Folder 32</dcterms:isPartOf>
    <dcterms:source>https://archivesspace.library.manoa.hawaii.edu/repositories/4/resources/15</dcterms:source>
    <dcterms:identifier>http://congressarchivesdev.lib.wvu.edu/record/10524/72356</dcterms:identifier>
    <edm:preview>https://evols.library.manoa.hawaii.edu/bitstreams/45b2b536-7e88-438f-9bd3-e016bbb85413/download</edm:preview>
    <edm:isShownAt>https:/evols.library.manoa.hawaii.edu/items/e4d5d07a-29d1-404d-baf3-6ac5e2f0242b</edm:isShownAt>
    <dc:type>speeches (documents)</dc:type>
    <dcterms:subject>Taxation; Economics and Public Finance</dcterms:subject>
    <dcterms:description/>
    <dcterms:type>Text</dcterms:type>
  </item>
  <item>
    <dcterms:provenance>Hawaiʻi Congressional Papers Collection, University of Hawaiʻi at Mānoa Library, Honolulu, HI</dcterms:provenance>
    <dcterms:title>Kiwanis Club of Honolulu</dcterms:title>
    <dcterms:date>February 8, 1983</dcterms:date>
    <dcterms:created>1983-02-08</dcterms:created>
    <dcterms:creator>Inouye, Daniel K., 1924-2012</dcterms:creator>
    <dcterms:rights>http://rightsstatements.org/vocab/NKC/1.0/</dcterms:rights>
    <dcterms:language>eng</dcterms:language>
    <dcterms:temporal>98th (1983-1985)</dcterms:temporal>
    <dcterms:relation>Senator Daniel K. Inouye Papers</dcterms:relation>
    <dcterms:isPartOf>MANUSCRIPT-HCPC00011, Speeches, Chronological, Box SP5, Folder 32</dcterms:isPartOf>
    <dcterms:source>https://archivesspace.library.manoa.hawaii.edu/repositories/4/resources/15</dcterms:source>
    <dcterms:identifier>http://congressarchivesdev.lib.wvu.edu/record/https://hdl.handle.net/10524/72356</dcterms:identifier>
    <edm:preview>https://evols.library.manoa.hawaii.edu/bitstreams/45b2b536-7e88-438f-9bd3-e016bbb85413/download</edm:preview>
    <edm:isShownAt>https:/evols.library.manoa.hawaii.edu/items/e4d5d07a-29d1-404d-baf3-6ac5e2f0242b</edm:isShownAt>
    <dc:type>speeches (documents)</dc:type>
    <dcterms:subject>Economics and Public Finance; Taxation</dcterms:subject>
    <dcterms:format>42 Pages</dcterms:format>
    <dcterms:description/>
    <dcterms:type>Text</dcterms:type>
  </item>
  <item>
    <dcterms:provenance>Hawaiʻi Congressional Papers Collection, University of Hawaiʻi at Mānoa Library, Honolulu, HI</dcterms:provenance>
    <dcterms:title>Lanai, Native Hawaiian Health ʻAha</dcterms:title>
    <dcterms:date>January 14, 1998</dcterms:date>
    <dcterms:created>1998-01-14</dcterms:created>
    <dcterms:creator>Inouye, Daniel K., 1924-2012</dcterms:creator>
    <dcterms:rights>http://rightsstatements.org/vocab/NKC/1.0/</dcterms:rights>
    <dcterms:language>eng</dcterms:language>
    <dcterms:temporal>105th (1997-1999)</dcterms:temporal>
    <dcterms:relation>Senator Daniel K. Inouye Papers</dcterms:relation>
    <dcterms:isPartOf>MANUSCRIPT-HCPC00011, Speeches, Chronological, Box SP11, Folder 37</dcterms:isPartOf>
    <dcterms:source>https://archivesspace.library.manoa.hawaii.edu/repositories/4/resources/15</dcterms:source>
    <dcterms:identifier>http://congressarchivesdev.lib.wvu.edu/record/10524/72603</dcterms:identifier>
    <edm:preview>https://evols.library.manoa.hawaii.edu/bitstreams/50024a00-1507-4efd-bab5-7dbcd9cd6606/download</edm:preview>
    <edm:isShownAt>https:/evols.library.manoa.hawaii.edu/items/1dc31362-5b27-46b4-83ff-7c041fe3c654</edm:isShownAt>
    <dc:type>speeches (documents)</dc:type>
    <dcterms:subject>Native Americans; Social Welfare; Health</dcterms:subject>
    <dcterms:description/>
    <dcterms:type>Text</dcterms:type>
  </item>
  <item>
    <dcterms:provenance>Hawaiʻi Congressional Papers Collection, University of Hawaiʻi at Mānoa Library, Honolulu, HI</dcterms:provenance>
    <dcterms:title>Lanai, Native Hawaiian Health ʻAha</dcterms:title>
    <dcterms:date>January 14, 1998</dcterms:date>
    <dcterms:created>1998-01-14</dcterms:created>
    <dcterms:creator>Inouye, Daniel K., 1924-2012</dcterms:creator>
    <dcterms:rights>http://rightsstatements.org/vocab/NKC/1.0/</dcterms:rights>
    <dcterms:language>eng</dcterms:language>
    <dcterms:temporal>105th (1997-1999)</dcterms:temporal>
    <dcterms:relation>Senator Daniel K. Inouye Papers</dcterms:relation>
    <dcterms:isPartOf>MANUSCRIPT-HCPC00011, Speeches, Chronological, Box SP11, Folder 37</dcterms:isPartOf>
    <dcterms:source>https://archivesspace.library.manoa.hawaii.edu/repositories/4/resources/15</dcterms:source>
    <dcterms:identifier>http://congressarchivesdev.lib.wvu.edu/record/https://hdl.handle.net/10524/72603</dcterms:identifier>
    <edm:preview>https://evols.library.manoa.hawaii.edu/bitstreams/50024a00-1507-4efd-bab5-7dbcd9cd6606/download</edm:preview>
    <edm:isShownAt>https:/evols.library.manoa.hawaii.edu/items/1dc31362-5b27-46b4-83ff-7c041fe3c654</edm:isShownAt>
    <dc:type>speeches (documents)</dc:type>
    <dcterms:subject>Health; Social Welfare; Native Americans</dcterms:subject>
    <dcterms:format>3 Pages</dcterms:format>
    <dcterms:description/>
    <dcterms:type>Text</dcterms:type>
  </item>
  <item>
    <dcterms:provenance>Hawaiʻi Congressional Papers Collection, University of Hawaiʻi at Mānoa Library, Honolulu, HI</dcterms:provenance>
    <dcterms:title>Sugar Club, NYC, Sugar: A Critical Juncture</dcterms:title>
    <dcterms:date>May 19, 1982</dcterms:date>
    <dcterms:created>1982-05-19</dcterms:created>
    <dcterms:creator>Inouye, Daniel K., 1924-2012</dcterms:creator>
    <dcterms:rights>http://rightsstatements.org/vocab/NKC/1.0/</dcterms:rights>
    <dcterms:language>eng</dcterms:language>
    <dcterms:temporal>97th (1981-1983)</dcterms:temporal>
    <dcterms:relation>Senator Daniel K. Inouye Papers</dcterms:relation>
    <dcterms:isPartOf>MANUSCRIPT-HCPC00011, Speeches, Chronological, Box SP5, Folder 9</dcterms:isPartOf>
    <dcterms:source>https://archivesspace.library.manoa.hawaii.edu/repositories/4/resources/15</dcterms:source>
    <dcterms:identifier>http://congressarchivesdev.lib.wvu.edu/record/10524/72333</dcterms:identifier>
    <edm:preview>https://evols.library.manoa.hawaii.edu/bitstreams/88a7b512-01de-4e7f-9e12-e74d6634e5ca/download</edm:preview>
    <edm:isShownAt>https:/evols.library.manoa.hawaii.edu/items/635af838-053b-4d0a-806e-60de62dcbb7d</edm:isShownAt>
    <dc:type>speeches (documents)</dc:type>
    <dcterms:subject>Commerce; Economics and Public Finance</dcterms:subject>
    <dcterms:description/>
    <dcterms:type>Text</dcterms:type>
  </item>
  <item>
    <dcterms:provenance>Hawaiʻi Congressional Papers Collection, University of Hawaiʻi at Mānoa Library, Honolulu, HI</dcterms:provenance>
    <dcterms:title>Sugar Club, NYC, Sugar: A Critical Juncture</dcterms:title>
    <dcterms:date>May 19, 1982</dcterms:date>
    <dcterms:created>1982-05-19</dcterms:created>
    <dcterms:creator>Inouye, Daniel K., 1924-2012</dcterms:creator>
    <dcterms:rights>http://rightsstatements.org/vocab/NKC/1.0/</dcterms:rights>
    <dcterms:language>eng</dcterms:language>
    <dcterms:temporal>97th (1981-1983)</dcterms:temporal>
    <dcterms:relation>Senator Daniel K. Inouye Papers</dcterms:relation>
    <dcterms:isPartOf>MANUSCRIPT-HCPC00011, Speeches, Chronological, Box SP5, Folder 9</dcterms:isPartOf>
    <dcterms:source>https://archivesspace.library.manoa.hawaii.edu/repositories/4/resources/15</dcterms:source>
    <dcterms:identifier>http://congressarchivesdev.lib.wvu.edu/record/https://hdl.handle.net/10524/72333</dcterms:identifier>
    <edm:preview>https://evols.library.manoa.hawaii.edu/bitstreams/88a7b512-01de-4e7f-9e12-e74d6634e5ca/download</edm:preview>
    <edm:isShownAt>https:/evols.library.manoa.hawaii.edu/items/635af838-053b-4d0a-806e-60de62dcbb7d</edm:isShownAt>
    <dc:type>speeches (documents)</dc:type>
    <dcterms:subject>Economics and Public Finance; Commerce</dcterms:subject>
    <dcterms:format>20 Pages</dcterms:format>
    <dcterms:description/>
    <dcterms:type>Text</dcterms:type>
  </item>
  <item>
    <dcterms:provenance>Hawaiʻi Congressional Papers Collection, University of Hawaiʻi at Mānoa Library, Honolulu, HI</dcterms:provenance>
    <dcterms:title>[Vietnam War conscientious objectors]</dcterms:title>
    <dcterms:date>May 26, 1968</dcterms:date>
    <dcterms:created>1968-05-26</dcterms:created>
    <dcterms:creator>Inouye, Daniel K., 1924-2012</dcterms:creator>
    <dcterms:rights>http://rightsstatements.org/vocab/NKC/1.0/</dcterms:rights>
    <dcterms:language>eng</dcterms:language>
    <dcterms:temporal>90th (1967-1969)</dcterms:temporal>
    <dcterms:relation>Senator Daniel K. Inouye Papers</dcterms:relation>
    <dcterms:isPartOf>MANUSCRIPT-HCPC00011, Speeches and messages, Box SP2, Folder 5</dcterms:isPartOf>
    <dcterms:source>https://archivesspace.library.manoa.hawaii.edu/repositories/4/resources/15</dcterms:source>
    <dcterms:identifier>http://congressarchivesdev.lib.wvu.edu/record/10524/71704</dcterms:identifier>
    <edm:preview>https://evols.library.manoa.hawaii.edu/bitstreams/6bb97ec2-7715-44b2-bb8d-4abbf824f3ef/download</edm:preview>
    <edm:isShownAt>https:/evols.library.manoa.hawaii.edu/items/996ee1ef-bc57-4003-9077-5ac62d15ca3c</edm:isShownAt>
    <dc:type>speeches (documents)</dc:type>
    <dcterms:subject>International Affairs; Armed Forces and National Security</dcterms:subject>
    <dcterms:description/>
    <dcterms:type>Text</dcterms:type>
  </item>
  <item>
    <dcterms:provenance>Hawaiʻi Congressional Papers Collection, University of Hawaiʻi at Mānoa Library, Honolulu, HI</dcterms:provenance>
    <dcterms:title>[Vietnam War conscientious objectors]</dcterms:title>
    <dcterms:date>May 26, 1968</dcterms:date>
    <dcterms:created>1968-05-26</dcterms:created>
    <dcterms:creator>Inouye, Daniel K., 1924-2012</dcterms:creator>
    <dcterms:rights>http://rightsstatements.org/vocab/NKC/1.0/</dcterms:rights>
    <dcterms:language>eng</dcterms:language>
    <dcterms:temporal>90th (1967-1969)</dcterms:temporal>
    <dcterms:relation>Senator Daniel K. Inouye Papers</dcterms:relation>
    <dcterms:isPartOf>MANUSCRIPT-HCPC00011, Speeches and messages, Box SP2, Folder 5</dcterms:isPartOf>
    <dcterms:source>https://archivesspace.library.manoa.hawaii.edu/repositories/4/resources/15</dcterms:source>
    <dcterms:identifier>http://congressarchivesdev.lib.wvu.edu/record/https://hdl.handle.net/10524/71704</dcterms:identifier>
    <edm:preview>https://evols.library.manoa.hawaii.edu/bitstreams/6bb97ec2-7715-44b2-bb8d-4abbf824f3ef/download</edm:preview>
    <edm:isShownAt>https:/evols.library.manoa.hawaii.edu/items/996ee1ef-bc57-4003-9077-5ac62d15ca3c</edm:isShownAt>
    <dc:type>speeches (documents)</dc:type>
    <dcterms:subject>International Affairs; Armed Forces and National Security</dcterms:subject>
    <dcterms:format>3 Pages</dcterms:format>
    <dcterms:description/>
    <dcterms:type>Text</dcterms:type>
  </item>
  <item>
    <dcterms:provenance>Hawaiʻi Congressional Papers Collection, University of Hawaiʻi at Mānoa Library, Honolulu, HI</dcterms:provenance>
    <dcterms:title>1960's Vietnam era youth's disenchantment w/America</dcterms:title>
    <dcterms:date>approximately 1960's</dcterms:date>
    <dcterms:created>1960/1969</dcterms:created>
    <dcterms:creator>Inouye, Daniel K., 1924-2012</dcterms:creator>
    <dcterms:rights>http://rightsstatements.org/vocab/NKC/1.0/</dcterms:rights>
    <dcterms:language>eng</dcterms:language>
    <dcterms:temporal>86th (1959-1961)</dcterms:temporal>
    <dcterms:relation>Senator Daniel K. Inouye Papers</dcterms:relation>
    <dcterms:isPartOf>MANUSCRIPT-HCPC00011, Speeches, Undated, Box SP1, Folder 1</dcterms:isPartOf>
    <dcterms:source>https://archivesspace.library.manoa.hawaii.edu/repositories/4/resources/15</dcterms:source>
    <dcterms:identifier>http://congressarchivesdev.lib.wvu.edu/record/10524/71360</dcterms:identifier>
    <edm:preview>https://evols.library.manoa.hawaii.edu/bitstreams/3ff20b22-11af-4b0a-89d9-282ee1fe6333/download</edm:preview>
    <edm:isShownAt>https:/evols.library.manoa.hawaii.edu/items/51f406d1-d5c5-41b2-8c86-6c2c8bff4196</edm:isShownAt>
    <dc:type>speeches (documents)</dc:type>
    <dcterms:subject>Armed Forces and National Security</dcterms:subject>
    <dcterms:description/>
    <dcterms:type>Text</dcterms:type>
  </item>
  <item>
    <dcterms:provenance>Hawaiʻi Congressional Papers Collection, University of Hawaiʻi at Mānoa Library, Honolulu, HI</dcterms:provenance>
    <dcterms:title>1960's Vietnam era youth's disenchantment w/America</dcterms:title>
    <dcterms:date>approximately 1960's</dcterms:date>
    <dcterms:created>1960/1969</dcterms:created>
    <dcterms:creator>Inouye, Daniel K., 1924-2012</dcterms:creator>
    <dcterms:rights>http://rightsstatements.org/vocab/NKC/1.0/</dcterms:rights>
    <dcterms:language>eng</dcterms:language>
    <dcterms:temporal>86th (1959-1961)</dcterms:temporal>
    <dcterms:relation>Senator Daniel K. Inouye Papers</dcterms:relation>
    <dcterms:isPartOf>MANUSCRIPT-HCPC00011, Speeches, Undated, Box SP1, Folder 1</dcterms:isPartOf>
    <dcterms:source>https://archivesspace.library.manoa.hawaii.edu/repositories/4/resources/15</dcterms:source>
    <dcterms:identifier>http://congressarchivesdev.lib.wvu.edu/record/https://hdl.handle.net/10524/71360</dcterms:identifier>
    <edm:preview>https://evols.library.manoa.hawaii.edu/bitstreams/3ff20b22-11af-4b0a-89d9-282ee1fe6333/download</edm:preview>
    <edm:isShownAt>https:/evols.library.manoa.hawaii.edu/items/51f406d1-d5c5-41b2-8c86-6c2c8bff4196</edm:isShownAt>
    <dc:type>speeches (documents)</dc:type>
    <dcterms:subject>Armed Forces and National Security</dcterms:subject>
    <dcterms:format>10 Pages</dcterms:format>
    <dcterms:description/>
    <dcterms:type>Text</dcterms:type>
  </item>
  <item>
    <dcterms:provenance>Hawaiʻi Congressional Papers Collection, University of Hawaiʻi at Mānoa Library, Honolulu, HI</dcterms:provenance>
    <dcterms:title>Praise for Democratic Party - Tribute to Gov. Jack Burns</dcterms:title>
    <dcterms:date>approximately 1963</dcterms:date>
    <dcterms:created>1963</dcterms:created>
    <dcterms:creator>Inouye, Daniel K., 1924-2012</dcterms:creator>
    <dcterms:rights>http://rightsstatements.org/vocab/NKC/1.0/</dcterms:rights>
    <dcterms:language>eng</dcterms:language>
    <dcterms:temporal>87th (1961-1963)</dcterms:temporal>
    <dcterms:relation>Senator Daniel K. Inouye Papers</dcterms:relation>
    <dcterms:isPartOf>MANUSCRIPT-HCPC00011, Speeches, Undated, Box SP1, Folder 1</dcterms:isPartOf>
    <dcterms:source>https://archivesspace.library.manoa.hawaii.edu/repositories/4/resources/15</dcterms:source>
    <dcterms:identifier>http://congressarchivesdev.lib.wvu.edu/record/10524/71361</dcterms:identifier>
    <edm:preview>https://evols.library.manoa.hawaii.edu/bitstreams/09aa5c15-e917-4dd3-bd69-5bd2637c91c4/download</edm:preview>
    <edm:isShownAt>https:/evols.library.manoa.hawaii.edu/items/81dc9b02-3d68-4267-8a10-003e8860aa91</edm:isShownAt>
    <dc:type>speeches (documents)</dc:type>
    <dcterms:subject>Government Operations and Politics</dcterms:subject>
    <dcterms:description/>
    <dcterms:type>Text</dcterms:type>
  </item>
  <item>
    <dcterms:provenance>Hawaiʻi Congressional Papers Collection, University of Hawaiʻi at Mānoa Library, Honolulu, HI</dcterms:provenance>
    <dcterms:title>Praise for Democratic Party - Tribute to Gov. Jack Burns</dcterms:title>
    <dcterms:date>approximately 1963</dcterms:date>
    <dcterms:created>1963</dcterms:created>
    <dcterms:creator>Inouye, Daniel K., 1924-2012</dcterms:creator>
    <dcterms:rights>http://rightsstatements.org/vocab/NKC/1.0/</dcterms:rights>
    <dcterms:language>eng</dcterms:language>
    <dcterms:temporal>87th (1961-1963)</dcterms:temporal>
    <dcterms:relation>Senator Daniel K. Inouye Papers</dcterms:relation>
    <dcterms:isPartOf>MANUSCRIPT-HCPC00011, Speeches, Undated, Box SP1, Folder 1</dcterms:isPartOf>
    <dcterms:source>https://archivesspace.library.manoa.hawaii.edu/repositories/4/resources/15</dcterms:source>
    <dcterms:identifier>http://congressarchivesdev.lib.wvu.edu/record/https://hdl.handle.net/10524/71361</dcterms:identifier>
    <edm:preview>https://evols.library.manoa.hawaii.edu/bitstreams/09aa5c15-e917-4dd3-bd69-5bd2637c91c4/download</edm:preview>
    <edm:isShownAt>https:/evols.library.manoa.hawaii.edu/items/81dc9b02-3d68-4267-8a10-003e8860aa91</edm:isShownAt>
    <dc:type>speeches (documents)</dc:type>
    <dcterms:subject>Government Operations and Politics</dcterms:subject>
    <dcterms:format>2 Pages</dcterms:format>
    <dcterms:description/>
    <dcterms:type>Text</dcterms:type>
  </item>
</items>
```

</details>

**Comment by mmizalmond** [comment]
Identifier field is pulling url from site and putting it together with identifier. Can we have only what appears in this field?

![Image](https://github.com/scientist-softserv/west-virginia-university/assets/148279275/c0cd9bbc-30a1-485c-9337-5761a962fc98)

**Comment by ShanaLMoore** [comment]
# QA RESULTS [REWORK]: PASS ✅ 

- [ ] identifier should not render a double url



![Image](https://github.com/scientist-softserv/west-virginia-university/assets/10081604/609e27b8-8de5-4866-8afe-84ed6e93f08e)

**Comment by ShanaLMoore** [comment]
❌ 

[Can we remove http://congressarchivesdev.liblwvu.edu/record/ from identifier in the export?](https://github.com/scientist-softserv/west-virginia-university/issues/93#issuecomment-1783364846)

<img width="380" alt="image" src="https://github.com/scientist-softserv/west-virginia-university/assets/10081604/4ec3933a-26c0-476d-acef-f81a5427ae1a">

**Comment by kirkkwang** [comment]
# QA RESULTS PASS ✅

<img width="873" alt="image" src="https://github.com/scientist-softserv/west-virginia-university/assets/19597776/f1d157ca-d8b3-42d5-8229-e05890278337">

**Comment by mmizalmond** [comment]
Beautiful! Thank you.
